### DISCUSSION TITLE
Multiple keypress responses?

### DISCUSSION THREAD

**Discussion by bashorkl** [jspsych/jsPsych#1447] [Q&A]
Hi, I'm new to the jsPsych online experiment builder and need some help. How do I record multiple keypress responses per trial (i.e. a double digit response per trial)? Thank you in advance!

**Comment by jodeleeuw** [comment]
Hi @bashorkl,

At the moment none of the built-in plugins allow you to do this. There are a few options:

1. You can use two consecutive `-keyboard-response` trials to capture the input. If you make the `stimulus` the same then the participant won't notice that the trial has "switched."
2. I believe that @kurokida's psychophysics plugin allows for this. See [here](https://github.com/jspsych/jsPsych/discussions/1128). 
3. You could modify a plugin file to allow for multiple keypresses.
4. You could use the `survey-text` plugin if you don't mind responses being typed into a form element.

**Reply by kurokida** [reply]
The keyboard_event.html sample program on [this page](https://jspsychophysics.hes.kyushu-u.ac.jp/demo_explanation.html) may be useful
